src/config.c

```diff
@@ -121,6 +121,23 @@ CONFIG_GETTER(getMaxDocTableSize) {
   return sdscatprintf(ss, "%lu", config->maxDocTableSize);
 }
 
+// MAXSEARCHRESULTS
+CONFIG_SETTER(setMaxSearchResults) {  
+  long long newsize = 0;
+  int acrc = AC_GetLongLong(ac, &newsize, 0);
+  CHECK_RETURN_PARSE_ERROR(acrc);
+  if (newsize == -1) {
+    newsize = UINT64_MAX;
+  }
+  config->maxSearchResults = newsize;
+  return REDISMODULE_OK;
+}
+
+CONFIG_GETTER(getMaxSearchResults) {
+  sds ss = sdsempty();
+  return sdscatprintf(ss, "%lu", config->maxSearchResults);
```

Comment:
So if we will set it to -1 and we will do ft.configget we will not see -1 no?

Reply:
was uint_max. changed to unlimited